frontend/src/container/IngestionSettings/IngestionSettings.tsx

```diff
@@ -3,18 +3,16 @@ import './IngestionSettings.styles.scss';
 import { Skeleton, Table, Typography } from 'antd';
 import type { ColumnsType } from 'antd/es/table';
 import getIngestionData from 'api/settings/getIngestionData';
+import { useAppContext } from 'providers/App/App';
 import { useQuery } from 'react-query';
-import { useSelector } from 'react-redux';
-import { AppState } from 'store/reducers';
 import { IngestionDataType } from 'types/api/settings/ingestion';
-import AppReducer from 'types/reducer/app';
 
 export default function IngestionSettings(): JSX.Element {
-	const { user } = useSelector<AppState, AppReducer>((state) => state.app);
+	const { user } = useAppContext();
 
 	const { data: ingestionData, isFetching } = useQuery({
 		queryFn: getIngestionData,
-		queryKey: ['getIngestionData', user?.userId],
+		queryKey: ['getIngestionData', user?.id],
```

Comment:
add enabled check here.

Reply:
will check in follow up PR , didn't update the existing code logics